projects/tsbp/index.html

```diff
@@ -0,0 +1,97 @@
+                            <p>Thomas Cohn, Odest Chadwicke Jenkins, Karhtik Desingh, Zhen Zeng</p>
+                        </header>
+
+                        <!-- Path to a pitch image. Can be removed if not needed. -->
+                        <!-- <div class="image pitch centered"><img src="../images/headshots/fetch.jpg" alt="" /></div> -->
+
+                        <p>
+                            We present Tangent Space Belief Propagation (TSBP) as a method for graph denoising to improve the robustness of manifold learning algorithms.  Dimension reduction by manifold learning relies heavily on the accurate selection of nearest neighbors, which has proven an open problem for sparse and noisy datasets. TSBP uses global nonparametric belief propagation to accurately estimate the tangent spaces of the underlying manifold at each data point. Edges of the neighborhood graph that deviate from the tangent spaces are then removed. The resulting denoised graph can then be embedded into a lower-dimensional space using methods from existing manifold learning algorithms. Artificially generated manifold data, simulated sensor data from a mobile robot, and high dimensional tactile sensory data are used to demonstrate the efficacy of our TSBP method.
+                        </p>
+
+                        <!-- Link to PDF. -->
+                        <p><a href="https://ieeexplore.ieee.org/document/9166624"><i class="fa fa-file-pdf"></i> PDF</a></p>
+
+
+                    </article>
+
+                    <article class="box post">
+                        <header>
+                            <h2>Video</h2>
+                        </header>
+
+                        <!-- Link to video on YouTube. -->
+                        <!-- To get your video link, go to your video and click "Share" -> "Embed" to get this iframe.
+                             You can remove width and height. -->
+                        <div class="video-container">
+                            <iframe class="video" src="https://www.youtube-nocookie.com/embed/fAFwI7ll3bg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                        </div>
+                    </article>
+
+                    <article class="box post">
+                        <header>
+                            <h2>Citation</h2>
+                        </header>
+
+                        <!-- The whitespace between <pre> tags will be preserved exactly how it is in the code. -->
+                        <div class="citation">
+<pre><code>@ARTICLE{9166624,
+  author={T. {Cohn} and O. C. {Jenkins} and K. {Desingh} and Z. {Zeng}},
+  journal={IEEE Robotics and Automation Letters}, 
+  title={TSBP: Tangent Space Belief Propagation for Manifold Learning}, 
+  year={2020},
+  volume={5},
+  number={4},
+  pages={6694-6701},}
+</code></pre>
```

Comment:
Do you want to have a more descriptive key, like `cohn2020tsbp`? Also you don't need to put last names in curly brackets. You should put the TSBP in curly brackets though. I've also had issues with trailing commas with some latex compilers.

Reply:
Just being picky because this is what people who cite you are supposed to copy / paste. :)

Reply:
Fixed